var/spack/repos/builtin/packages/fms/package.py

```diff
@@ -59,6 +59,11 @@ class Fms(CMakePackage):
         git = "https://github.com/JCSDA/fms.git"
     # *DH 20220602
 
+    variant(
```

Comment:
Instead of this we should pull in  the updates from https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/fms/package.py (can be a separate PR so you don't have to deal with it)

Reply:
I can do this if you like

Reply:
@climbfuji If you can, please. I have a [PR in to spack mainline](https://github.com/spack/spack/pull/35968) to update some other GFE packages. 

It's also possible we might want to wait for the [ESMF PR from @theurich](https://github.com/spack/spack/pull/35980) before pulling in spack? Not sure.

Reply:
Also, I will say I tried using the newer spack `maintainers()` function in this PR but spack wasn't happy. It's possible my spack was old?

Reply:
We haven't yet pulled that functionality from main spack, so with NOAA-EMC spack it will yell at you about the function being undefined :)